### PR TITLE
SEO-171911 - JS Missing H1

### PR DESCRIPTION
**Pull request by Sharly-Atieno**
Hi @Aishwarya-Ganesan,
Please review the changes for missing H1 below.

Regards,
Sharly A.

### CODE REVIEW

File: js/WebAPI/DocumentEditor.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ keywords: documenteditor,syncfusion,documenteditor webapi
 api: /api/js/ejdocumenteditor
 ---
 
-## Import
```
Comment (gowthamanthavasiyappan) [inline js/WebAPI/DocumentEditor.md:11]: Check meta details and h1 tag will be above 20 to below 70 characters.

**Review by gowthamanthavasiyappan — CHANGES_REQUESTED** [review]
Commented

File: js/WebAPI/DocumentEditor.md (hunk 1 above)
Comment (Sharly-Atieno) [reply js/WebAPI/DocumentEditor.md:11]: Done

**Review by gowthamanthavasiyappan — APPROVED** [review]
Approved

File: js/WebAPI/DocumentEditor.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: webAPI reference for DocumentEditor
-description: webAPI reference for 
+title: WebAPI Reference in JavaScript DocumentEditor Control | Syncfusion
+description: Learn here about webapi reference support in Syncfusion Essential JavaScript DocumentEditor Control, its elements, and more.
```
Comment (Girija-Rajendran) [inline js/WebAPI/DocumentEditor.md:4]: Learn here about WebAPI........................

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**